src/main.js

```diff
@@ -2,6 +2,8 @@ import { library as fortawesome } from '@fortawesome/fontawesome-svg-core'
 
 import * as icons from '@/utils/font-awesome'
 import { createCore } from '@/core'
+// TODO: add .scss in list of allowed extensions in eslint-config-icij
```

Comment:
Todo now or later?

Reply:
I will do it following this yes, but it needs for a new version of the package to be released. 

Do you think this single comment is a reason to block the merge of this PR with 'request changes' ? NB: you can stack your comments in a pull request review instead of making a new review for each comment. You can also comment without approving nor request changes if you don't want to approve but don't need for the next round of review to necessarily be made by you.